core/lib/dal/src/storage_web3_dal.rs

```diff
@@ -237,17 +237,17 @@ impl StorageWeb3Dal<'_, '_> {
         Ok(row.map(|row| row.bytecode))
     }
 
-    /// This method doesn't check if block with number equals to `block_number`
-    /// is present in the database. For such blocks `None` will be returned.
-    pub async fn get_factory_dep_unchecked(
+    /// This method doesn't check if block with number equals to `block_number` is present in the database.
+    pub async fn get_factory_dep_and_miniblock_unchecked(
         &mut self,
         hash: H256,
         block_number: MiniblockNumber,
```

Comment:
I wonder whether it would make sense to get rid of this arg, given that we filter by the output `MiniblockNumber` in any case. Not sure what the access patterns for these calls are, but theoretically this could allow to cache some deps earlier.

Reply:
Yeah, we can get rid of it. It will be cleaner but in terms of performance it won't change anything because I'm almost sure that in 99.99% `inserted_at <= storage.miniblock_number` is true. I will get rid of it anyway